simulation/g4detectors/SQDigitizer.cc

```diff
@@ -160,7 +160,7 @@ void SQDigitizer::digitizePlane(const std::string& detName)
     const PHG4Hit& g4hit = *(it->second);
 
     int track_id = g4hit.get_trkid();
-    if(track_id < 0) continue;         //only save primary track hits
+    //if(track_id < 0) continue;         //only save primary track hits //WPM
```

Comment:
Can you comment on why was this needed for your studies? There was actually a recent change that introduces a `digitize_secondaries` flag and does not apply this cut when that flag is on - so I will modify the code to behave like this. But would be good to know in which cases we want to digitize the secondaries.

Reply:
So for particle guns, if there's any showering of the initial particle, any hits associated with secondaries are not included in the tracker hit record.  One thing that we're trying to look at is particle ID, so if there's e.g. a pair production from a photon or brem from an electron, then tracks won't get built for the secondary particles unless the secondaries are digitized.  Hits for the secondaries do occur in the ECal even if only primary track hits are saved, but not in the tracker

Reply:
Ok thanks, this is useful to know! For now, we will keep this `digitize_secondaries` function in the merged PR, and maybe we can set it to always `True` in our `Fun4Sim.C` macro.